database_mongo/seed.js

```diff
@@ -260,15 +273,36 @@ const seed = () => {
   return docsArray;
 };
 
+
+//create seed data chunk
+const seedData = seed();
```

Comment:
A case could be made for exporting this seed function, moving your addMany... database-specific functions each to their own files, then importing it over there, for modularity.